moneyadmin.sln

```diff
@@ -31,7 +31,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MoneyAdmin.WebApi", "src\Mo
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "6 - Tests", "6 - Tests", "{4061DB77-6E88-4756-B75F-E7CA6662EA2A}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MoneyAdmin.Domain.UnitTests", "src\Tests\MoneyAdmin.Domain.UnitTests\MoneyAdmin.Domain.UnitTests.csproj", "{886C722C-0CF0-4851-9438-E894F220FDBB}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MoneyAdmin.Domain.UnitTests", "src\Tests\MoneyAdmin.Domain.UnitTests\MoneyAdmin.Domain.UnitTests.csproj", "{886C722C-0CF0-4851-9438-E894F220FDBB}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MoneyAdmin.WebApi.Test", "src\MoneyAdmin.WebApi.Test\MoneyAdmin.WebApi.Test.csproj", "{ED26CAA2-71B3-42C6-9603-0FBD958571A1}"
```

Comment:
The project name must be **MoneyAdmin.WebApi.UnitTests** and the project path must be **src\MoneyAdmin.WebApi.UnitTest\MoneyAdmin.WebApi.UnitTests.csproj**